src/vscode-dts/vscode.proposed.chatParticipant.d.ts

```diff
@@ -249,6 +253,11 @@ declare module 'vscode' {
 		 */
 		readonly range: [start: number, end: number];
 
+		/**
+		 * A description of this value that could be used in an LLM prompt.
+		 */
+		readonly modelDescription?: string;
+
 		/**
 		 * The value of this reference. The `string | Uri | Location` types are used today, but this could expand in the future.
 		 */
```

Comment:
Let's think about a type-safe way of getting value types, like editor selection vs file-reference etc